include/veriblock/entities/btcblock.hpp

```diff
@@ -113,6 +111,9 @@ struct BtcBlock {
   uint32_t timestamp = 0;
   uint32_t bits = 0;
   uint32_t nonce = 0;
+
+ private:
+  static const std::string _name;
```

Comment:
I think it is safe to have it public.

Reply:
Yes and no. We should use only `::name()` static method to get this field